src/components/admin/button-tooltip/ButtonTooltip.tsx

```
@@ -92,19 +49,16 @@ export const ButtonTooltip = ({ children, position = 'bottom', offset = 8 }: But
-                        left: `${tooltipPosition.left}px`,
-                    }}
-                >
-                    {children}
-                </div>
+                    id={tooltipId}
+                    position={position}
+                    offsetInPixels={8}
+                    customMaxWidthInPixels={400}
+                    allowClickThrough={true}
+                    isCentered={true}
+                    children={children}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)